packages/cubejs-schema-compiler/test/integration/postgres/sql-generation.test.ts

```diff
@@ -292,9 +293,12 @@ describe('SQL Generation', () => {
 
     cube('visitor_checkins', {
       sql: \`
-      select * from visitor_checkins WHERE
-      \${FILTER_PARAMS.visitor_checkins.created_at.filter('created_at')} AND
-      \${FILTER_GROUP(FILTER_PARAMS.visitor_checkins.created_at.filter("(created_at - INTERVAL '3 DAY')"), FILTER_PARAMS.visitor_checkins.source.filter('source'))}
+      select visitor_checkins.* from visitor_checkins left join visitors on visitor_checkins.visitor_id = visitors.id WHERE
```

Comment:
Can you explain the change? Could you tell me why you didn't create another cube/test or why this one doesn't suit your needs?

Reply:
This is an fix of an incorrect data model for tests. `SECURITY_CONTEXT.filter` should work regardless of whether the cube has built a joiner for a particular querier or not.  Therefore, this filter must be written directly in the sql of the cube. This is one of the reasons why SECURITY_CONTEXT is deprecated. In the case of tesseract it led to the fact that in one of the tests fails, because tesseract knows how to omit unnecessary joins in subqueries. But this is not the error of tesseract, but of incorrect use of SECURITY_CONTEXT in the model.